web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -1309,6 +1311,60 @@ class AppVersionHistory extends Component {
     });
   };
 
+  handleActionButtonClicked = ({ versionLabel, sequence }) => {
+    if (this.props.isHelmManaged) {
+      this.setState({
+        showHelmDeployModalForVersionLabel: versionLabel,
+        showHelmDeployModalForSequence: sequence,
+      });
+    }
+  };
+
+  deployButtonStatus = (version) => {
```

Comment:
just copied this logic over from the other component for now. Need to refactor the `/apps` and `/versions` calls so that we add this stuff to the backend state as a selector.